src/commands/drain.js

```diff
@@ -25,16 +25,22 @@ async function list (params) {
   }
 
   drains.forEach((drain) => {
-    const { id, state, target: { url, drainType } } = drain;
-    Logger.println(`${id} -> ${state} for ${url} as ${drainType}`);
+    const { id, state, target } = drain;
+    const { url, drainType, indexPrefix } = target;
+
+    let drainView = `${id} -> ${state} for ${url} as ${drainType}`;
+    if (indexPrefix != null) {
+      drainView += `, index: \`${indexPrefix}-<YYYY-MM-DD>\``;
```

Comment:
is it an index name, or ... an index name template ?

Reply:
What do you mean?